keps/sig-api-machinery/3285-subresource-finalizers/README.md

```diff
@@ -0,0 +1,784 @@
+- `pkg/registry/node/runtimeclass/storage` : `2022/6/20` - 
+- `pkg/registry/policy/poddisruptionbudget/storage` : `2022/6/20` - 62.5%
+- `pkg/registry/policy/podsecuritypolicy/storage` : `2022/6/20` - 75.0%
+- `pkg/registry/rbac/clusterrole/storage` : `2022/6/20` -   
+- `pkg/registry/rbac/clusterrolebinding/storage` : `2022/6/20` -   
+- `pkg/registry/rbac/role/storage` : `2022/6/20` - 
+- `pkg/registry/rbac/rolebinding/storage` : `2022/6/20` - 
+- `pkg/registry/scheduling/priorityclass/storage` : `2022/6/20` - 83.3%
+- `pkg/registry/storage/csidriver/storage` : `2022/6/20` - 85.7%
+- `pkg/registry/storage/csinode/storage` : `2022/6/20` - 85.7%
+- `pkg/registry/storage/csistoragecapacity/storage` : `2022/6/20` - 85.7%
+- `pkg/registry/storage/storageclass/storage` : `2022/6/20` - 87.5%
+- `pkg/registry/storage/volumeattachment/storage` : `2022/6/20` - 66.7%
+- `staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource`: 2022/6/20` - 72.6%
+- `staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresourcedefinition`: 2022/6/20` - 13.1%
+
+CLI:
+- `staging/src/k8s.io/kubectl/pkg/cmd/get`: 2022/6/20` - 81.0%
+- `staging/src/k8s.io/kubectl/pkg/cmd/patch`: 2022/6/20` - 57.1%
+
+##### Integration tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+- TestFinalizeAllResources (similar tests to those of [scale](https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/apply/scale_test.go)): <link to test coverage>
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
+- Verify finalizers can be read through GET call to finalizers if getting finalizers is allowed:  <link to test coverage>
+- Verify finalizers can't be read through GET call to finalizers if getting finalizers isn't allowed:  <link to test coverage>
+- Verify resource can't be read through GET call to the resource if only getting finalizers is allowed:  <link to test coverage>
+- Verify finalizers can be updated through PATCH call to finalizers if updating finalizers is allowed:  <link to test coverage>
+- Verify finalizers can't be updated through PATCH call to finalizers if updating finalizers isn't allowed:  <link to test coverage>
+- Verify resource can't be updated through PATCH call to the resource if only updating finalizers is allowed:  <link to test coverage>
```

Comment:
Verify other metadata fields can not be updated

Reply:
Added.